### PR TITLE
Add preact-signals to labs page

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

File: packages/lit-dev-content/site/docs/v3/libraries/labs.md [hunk 1]
```diff
@@ -217,6 +217,20 @@ A plugin for [Eleventy](https://www.11ty.dev) that pre-renders Lit components at
 <tr>
 <td>
 
+[preact-signals](https://www.npmjs.com/package/@lit-labs/preact-signals)
+
+</td>
+<td>Preact Signals integration for Lit.</td>
+<td class="labs-table-links">
+
+[📄&nbsp;Docs](https://github.com/lit/lit/tree/main/packages/labs/preact-signals#readme "Docs")<br>[💬&nbsp;Feedback](https://github.com/lit/lit/discussions/4115 "Feedback")<br>[🐞&nbsp;Issues](https://github.com/lit/lit/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+%5Blabs%2Fpreact-signals%5D "Issues")
```
Comment (AndrewJakubowicz) [inline packages/lit-dev-content/site/docs/v3/libraries/labs.md:226]: I think the readme link needs to point to 3.0, since it's not in main.

Comment (justinfagnani) [reply packages/lit-dev-content/site/docs/v3/libraries/labs.md:226]: oh right, then this will have to be updated...

Comment (justinfagnani) [reply packages/lit-dev-content/site/docs/v3/libraries/labs.md]: done

**Review by AndrewJakubowicz — APPROVED** [review]
Nice!

**Pull request merged**